tendrl/commons/objects/node/flows/stop_services/__init__.py

```diff
@@ -11,14 +11,24 @@ def run(self):
         super(StopServices, self).run()
         services = self.parameters['Services[]']
         for service in services:
+            logger.log(
+                "info",
+                NS.publisher_id,
+                {
+                    "message": "Stopping service %s on node %s" %
+                               (service, NS.node_context.node_id)
+                },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id'],
+            )
             srv = NS.tendrl.objects.Service(service=service)
             if not srv.running:
                 logger.log(
                     "debug",
                     NS.publisher_id,
                     {
                         "message": "%s not running on "
-                        "%s" % (service, NS.node_context.fqdn)
+                        "%s" % (service, NS.node_context.node_id)
```

Comment:
use fqdn in all unmanage log message and job updates, node_id is not readable by users. I saw all the place node_id is used in unmanage flows and atoms

Reply:
import cluster flow is using fqdn is everywhere better we use fqdn in unmanage flow also

Reply:
In unmanage cluster, most of the places node_id is used. Stop services comes under unmanage. That is why node_id is used to maintain uniformity in unmanage flow.

Reply:
i feel we need to change all places in unmanage flow. @shtripat

Reply:
Please use fqdn only.